app/controllers/application_controller.rb

```diff
@@ -0,0 +1,86 @@
+require 'rubygems'
+require 'bundler/setup'
+require 'sinatra/base'
+
+Bundler.require(:default) 
+
+configure :test do
+ set :database, 'postgresql://localhost/test'
+ DB = Sequel.connect("postgresql://localhost/test")
+end
+
+configure :development do
+  set :database, 'postgresql://localhost/users'
+  DB = Sequel.connect("postgresql://localhost/users")
```

Comment:
Change the name of the db to `crud_development` instead of `users`. `users` is usually used to represent a table. For db_names, we usually use the name of the app and the env we use the db in.